types/applepayjs/index.d.ts

```diff
@@ -455,6 +455,11 @@ declare namespace ApplePayJS {
          * The payment pass object currently selected to complete the payment.
          */
         paymentPass: ApplePayPaymentPass;
+
+        /**
+         * The billing contact selected by the user for this transaction.
```

Comment:
I'd suggest matching the other members and using the exact text from the Apple documentation.

```suggestion
         * The billing contact associated with the card.
```

Reply:
Definitely, thanks!